root/dashboard/swag-f2b.py

```diff
@@ -4,7 +4,22 @@
 
 con = sqlite3.connect("/config/fail2ban/fail2ban.sqlite3")
 cur = con.cursor()
-results = cur.execute("SELECT jails.name, COUNT(bans.ip) AS bans FROM jails LEFT JOIN bans ON jails.name=bans.jail GROUP BY jails.name").fetchall()
+results = cur.execute("""
+    SELECT jails.name, 
+    COUNT(bans.ip) AS bans,
```

Comment:
@quietsy and @aptalca I was having a look to this PR as I was confused by what the dashboard was showing me: 8 bans, when there is none technically speaking:
![image](https://user-images.githubusercontent.com/11942518/152249259-8c66f8ac-1d7c-4777-a263-ed05140ce301.png)
I now better understand that bans is a count on table entries, but these entries may be unbanned at the time of this check, which was my case.
Dashboard vs SQLDBLite:
<img width="591" alt="image" src="https://user-images.githubusercontent.com/11942518/152249391-aa507caf-8abe-487a-b0d2-0770bd16cc07.png">

I wanted to submit an issue but I am not so sure if this is here a bug or a feature. I would find "Active Bans" (timeofban + bantime > now) more useful for users than "Bans". Maybe both could be useful, I honestly don't know, and as you seem to be both involved in this topic more than I am, I would like your opinion on it.

Many thanks for your contributions on the dashboard mod, it changed the way I am monitoring my swag container and helps me to focus my attention on important information I may have miss otherwise 👍